src/test/java/com/snowflake/kafka/connector/internal/streaming/SnowflakeSinkServiceV2IT.java

```diff
@@ -370,7 +370,59 @@ public void testStreamingIngest_multipleChannelPartitions() throws Exception {
   }
 
   @Test
-  public void testStreamingIngest_startTasksWithMultipleChannelPartitions() throws Exception {
+  public void testStreamingIngest_multipleChannelPartitionsWithTopic2Table() throws Exception {
+    final int partitionCount = 5;
+    final int recordsInEachPartition = 2;
+
+    Map<String, String> config = TestUtils.getConfForStreaming();
+    SnowflakeSinkConnectorConfig.setDefaultValues(config);
+    Map<String, String> topic2Table = new HashMap<>();
+    topic2Table.put(topic, table);
```

Comment:
can you put multiple topics here and then insert records for a topic partition not present in topic2table.

Reply:
Sounds good, added.